lib/compat.php

```diff
@@ -167,17 +167,18 @@ function register_block_type_from_metadata( $file_or_folder, $args = array() ) {
 
 		$settings          = array();
 		$property_mappings = array(
-			'title'       => 'title',
-			'category'    => 'category',
-			'context'     => 'context',
-			'parent'      => 'parent',
-			'icon'        => 'icon',
-			'description' => 'description',
-			'keywords'    => 'keywords',
-			'attributes'  => 'attributes',
-			'supports'    => 'supports',
-			'styles'      => 'styles',
-			'example'     => 'example',
+			'title'           => 'title',
+			'category'        => 'category',
+			'parent'          => 'parent',
+			'icon'            => 'icon',
+			'description'     => 'description',
+			'keywords'        => 'keywords',
+			'attributes'      => 'attributes',
+			'providesContext' => 'provides_context',
+			'usesContext'     => 'uses_context',
```

Comment:
Do we need to keep `'context'     => 'context'` for compat until 8.6?

Reply:
Good catch, for the plugin 👍 
I will add a line with a comment.